mudlet-mapper.xml

```diff
@@ -222,40 +222,38 @@ end</script>
             <Trigger isActive="yes" isFolder="no" isTempTrigger="no" isMultiline="no" isPerlSlashGOption="no" isColorizerTrigger="no" isFilterTrigger="no" isSoundTrigger="no" isColorTrigger="no" isColorTriggerFg="no" isColorTriggerBg="no">
                 <name>Alertness (Achaea+Imperian)</name>
                 <script>mmp.alertness = mmp.alertness or {}
-
+cecho(&quot;&lt;red&gt;\n&quot;..matches[4])
```

Comment:
Uh yea totally forgot about that. Will edit it once I can figure out github again.